proxy/operator_resource/resource_manager.py

```diff
@@ -122,6 +129,61 @@ def free_resource(self, is_good_resource: bool, op_resource: OpResourceModel) ->
                 _LOG.debug("disable resource: %s", op_resource)
                 op_signer.disabled_holder_list.append(op_holder)
 
+            try:
+                is_valid, spending_tokens_balance = await self._validate_op_balance_impl(op_signer)
+
+                if not is_valid:
+                    if disabled_op_signer := self._active_signer_dict.pop(op_signer.owner, None):
+                        self._disabled_signer_dict[disabled_op_signer.owner] = disabled_op_signer
+
+                # Earned tokens balance
+
+                evm_cfg = await self._server.get_evm_cfg()
+                neon_address = NeonAccount.from_raw(op_resource.eth_address, op_resource.chain_id)
```

Comment:
Please, validate that `op_resource.chain_id` `is not None`.
In the previous comment, I highlighted that the `chain_id` can be None.

Reply:
Same as above.